.github/workflows/build-and-run-smoke-tests.yml

```diff
@@ -3,23 +3,38 @@
 # | Edit _templates/dev.yml.erb instead.          |
 # +-----------------------------------------------+
 
-name: Dev
+name: Build and run smoke tests
 
 on:
   workflow_dispatch:
-  push:
+  workflow_run:
+    workflows: [Build Base Images]
+    types: [completed]
```

Comment:
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_run